pages/login.js

```diff
@@ -0,0 +1,100 @@
+import { useState } from 'react';
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+
+import { Layout } from '../components/Layout';
+import { useAuth } from '../contexts/AuthContext';
+import { FormInput } from '../components/Form/FormInput';
+
+const Login = () => {
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState();
+
+  const router = useRouter();
+  const { login, currentUser, logout } = useAuth();
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    setLoading(true);
+    setError('');
+    try {
+      await login(e.target.email.value, e.target.passwd.value);
+      router.push('/');
+    } catch (err) {
+      if (
+        err.code === 'auth/wrong-password' ||
+        err.code === 'auth/user-not-found'
+      ) {
```

Comment:
This code is absolutely fine. If we would get to more error codes you could use this trick -> 

```['errorcode1', 'errorcode2',  'errorcode3, ].includes(err.code)```